README.md

```diff
@@ -239,13 +239,22 @@ const clean = sanitizeHtml(dirty, {
 ```
 
 Similar to `allowedAttributes`, you can use `*` to allow classes with a certain prefix, or use `*` as a tag name to allow listed classes to be valid for any tag:
+
 ```js
 allowedClasses: {
   'code': [ 'language-*', 'lang-*' ],
   '*': [ 'fancy', 'simple' ]
 }
 ```
 
+Furthermore, regular expressions are supported too and **they must begin with `^` and end with `$`** otherwise the regex has no effect:
```

Comment:
The code changes look good as does the example below. I don't think it is appropriate to say they must begin with `^` and end with `$`. it is up to developers to decide how to handle this. For instance it could be very reasonable to allow any class name that starts with a certain prefix using `^` but not `$`, and that could be an effective way to prevent surprising effects depending on how you're controlling how CSS is inserted into the environment.

Reply:
The reason that I wrote it so opinionated is that a wildcard in class name (or allowing a certain prefix as you described) can lead to a XSS attack (at least in AngularJS).

I can remove `and **they must begin with `^` and end with `$`** otherwise the regex has no effect:` and add after the code example this:
> Note: It is strongly advised that the regular expressions begin with `^` and end with `$` so you explicitly declare what classes you allow and prevent security issues.

Reply:
Hmm, are you saying that an allowed class name in sanitize-html isn't necessarily escaped properly as an attribute? I think it would be, even if it contained a quote.

Reply:
found out that it doesn't escape `;`
![image](https://user-images.githubusercontent.com/22919198/141169012-061f5664-a4b3-46e6-877f-768924463d84.png) - will create an issue.

Meanwhile I updated readme suggesting to explicitly declare regExps.